components/button/button.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { Children } from "react";
```

Comment:
can we remove this `{ Children }` bit from line 1 since it's not being used anywhere?